<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Dibujar_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>571</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Agregar</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>511</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>UI - Neuronas</string>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Voltaje</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Posicion_X</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Posicion_Y</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>RED</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>GREEN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>113</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>BLUE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Captura">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>350</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Capturar</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="Id">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="Posicionx">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>120</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="PosicionY">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="Voltaje">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>62</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="red">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="green">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>240</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="blue">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>290</y>
         <width>42</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="res">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>191</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mostrar</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>361</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar Tabla</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Grafichview">
     <attribute name="title">
      <string>Dibujar</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>461</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Dibujar">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>350</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dibujar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Limpiar">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>350</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpne"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionOpne">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
